tests/e2e/default/contacts/add-new-district.wdio-spec.js

```diff
@@ -1,83 +1,78 @@
-const commonElements = require('@page-objects/default/common/common.wdio.page.js');
-const contactPage = require('@page-objects/default/contacts/contacts.wdio.page.js');
 const utils = require('@utils');
 const sentinelUtils = require('@utils/sentinel');
+const placeFactory = require('@factories/cht/contacts/place');
+const personFactory = require('@factories/cht/contacts/person');
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
 const loginPage = require('@page-objects/default/login/login.wdio.page');
 
+
 describe('Add new district tests : ', () => {
   before(async () => await loginPage.cookieLogin());
   afterEach(() => sentinelUtils.waitForSentinel());
```

Comment:
Please change this so retries are possible.
```suggestion
  afterEach(async () => {
    sentinelUtils.waitForSentinel();
    await utils.revertDb([/^form:/], true);
  });
```